src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +56,40 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+}
+
+/**
+ * Takes in a Person's address in the address book by block, street, unit, postal code
+ */
+
+public interface Address{
+    public String userInput();
+}
+
```

Comment:
You have to think here as an address has a block, street, unit and a postal code. So define the address as a combination of these. Remember to use encapsulation when defining each class.